templates/nerdgraphclient/integration_test.go.tmpl

```diff
@@ -1,3 +1,4 @@
+// Code generated by tutone: DO NOT EDIT
```

Comment:
Should we remove the `DO NOT EDIT` part?

Reply:
Yes, that comment could be misleading.